apps/tsri/src/sitemap.ts

```diff
@@ -2,40 +2,72 @@ import {ApiV1, generateSitemap} from '@wepublish/website/server'
 import {NextApiRequest} from 'next'
 import getConfig from 'next/config'
 import process from 'node:process'
+import {groupBy} from 'ramda'
 
 export const getSitemap = async (req: NextApiRequest): Promise<string> => {
   const siteUrl = process.env.WEBSITE_URL || ''
 
   const generate = generateSitemap({
     siteUrl,
-    title: 'Tsri'
+    title: 'Bajour'
```

Comment:
Why would it change to Bajour in a Tsri project?

Reply:
Oh this was from debugging api v1/v2, discarding these changes